src/core/viewports/viewport.js

```diff
@@ -303,7 +303,15 @@ export default class Viewport {
     return false;
   }
 
-  getDistanceScales() {
+  getDistanceScales(coordinateOrigin) {
```

Comment:
Nit: Changing the function declaration to `getDistanceScales(coordinateOrigin = null)` makes it clearer that this method can be called without an origin.